numba/tests/test_parfors.py

```diff
@@ -3739,6 +3740,78 @@ def find_maxima_3D_jit(args):
             find_maxima_3D_jit.py_func(args),
         )
 
+    @skip_parfors_unsupported
```

Comment:
Suggest all these tests go into a separate module `numba.tests.test_parallel_scheduling` or similar.

Reply:
Hmm....How about just a different high-level test section in the existing test_parfors.py file since it is parfor related?

Reply:
Given the parfors only nature of this I agree with not moving it in hindsight. Having a new test class might be a good idea just purely for the purposes of grouping similar things.